### PR TITLE
Open email Story 3 

### PR DESCRIPTION
**Pull request by vivhayles**
Please review our code?

### CODE REVIEW

File: src/Pages/OpenEmailPage/index.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { BrowserRouter, Route, Routes, Router } from "react-router-dom"
```
Comment (ashleycoles) [inline src/Pages/OpenEmailPage/index.jsx]: You can delete this file now, don't think it does anything

File: src/Components/OpenEmails/index.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react"
```
Comment (ashleycoles) [inline src/Components/OpenEmails/index.jsx]: Really this is your page now, can you move this into the pages folder please :)

File: src/Components/CloseEmailButton/index.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { Link } from "react-router-dom"
+
+function CloseEmailButton (){
```
Comment (ashleycoles) [inline src/Components/CloseEmailButton/index.jsx]: Any way you can make this more re-usable? What if it was a generic button with props that allow you to set the link and the text?

File: src/Components/CloseEmailButton/index.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    return (
+        <div className="grid justify-items-end pt-4">
+            <button className="px-4 py-1.5 bg-gray-500 text-white rounded hover:bg-gray-600">
```
Comment (ashleycoles) [inline src/Components/CloseEmailButton/index.jsx]: You probably don't need this button tag anymore, just move the classes onto the Link and go with that

File: src/Components/CloseEmailButton/index.jsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import { Link } from "react-router-dom"
+
+function CloseEmailButton (){
+    return (
+        <div className="grid justify-items-end pt-4">
```
Comment (ashleycoles) [inline src/Components/CloseEmailButton/index.jsx]: Having this container here also limits the usability of this component - I would remove it from here and instead put this code (just this div containing the button) into your OpenEmails page instead.

That way you can use this component anytime you need a button that links to a page.

**Review by ashleycoles — CHANGES_REQUESTED** [review]
Excellent work with this. Just a few minor comments

**Review by ashleycoles — APPROVED** [review]

File: src/Components/EmailSummary/index.jsx [hunk 6]
```diff
@@ -1,6 +1,12 @@
-function EmailSummary ({name, subject, body, date, read}) {
+import { Link, useSubmit } from "react-router-dom"
+import OpenEmails from "../../Pages/OpenEmailPage"
+
+function EmailSummary ({id, name, subject, body, date, read}) {
+            
+   
     return (
         <div className={`p-4 border-b border-gray-400 border-solid flex justify-between h-25" ${read == 1 ? "bg-header-grey text-white " : ""}`}>
```
Comment (ashleycoles) [inline src/Components/EmailSummary/index.jsx]: Just spotted there is an extra quote on this line - the one after h-25 can be deleted

File: src/Components/EmailSummary/index.jsx [hunk 7]
```diff
@@ -1,6 +1,12 @@
...
+            <Link to={`/emails/${id}`} onClick={<OpenEmails />}>
```
Comment (ashleycoles) [inline src/Components/EmailSummary/index.jsx]: You should move this Link to be the parent element of this component - at the momeny only the text is clickable, but making it the parent will make the whole 'row' clickable

**Review by ashleycoles — CHANGES_REQUESTED** [review]

**Comment by michaelday1991** [comment]
All changes resolved. Ready to merge to Main! @ashleycoles

**Review by ashleycoles — APPROVED** [review]

**Pull request merged**